tasks/configure.yml

```diff
@@ -1,12 +1,4 @@
 ---
-- name: Ensure ccache cache directory exists
```

Comment:
The commit message body of 58cbfd4780d448d8454a2ecc6d56fd5e9e0bb691 mentions "duplicating data", but this commit is about removing a task that might not work (if `ccache_cache_dir` is not defined) and is not required for ccache to work.